library/common/main_interface.h

```diff
@@ -11,12 +11,19 @@
 extern "C" { // functions
 #endif
 
+/**
+ * Initialize an underlying HTTP stream.
+ * @param engine, handle to the engine that will manage this stream.
+ * @return envoy_stream_t, handle to the underlying stream.
+ */
+envoy_stream_t init_stream(envoy_engine_t);
```

Comment:
Should the comment also say that the stream must be started before other stream operations happen? Is that confusing?

Reply:
sure

Reply:
I don't think it's particularly confusing, but I do agree it's worth calling out explicitly. Updated.